server/cellvs2_test.go

```diff
@@ -0,0 +1,553 @@
+				t.Errorf("%q. CellIDV2() = \n***%v***\n,\nwant\n***%v***", tt.name, string(body), tt.wants.body)
+			}
+		})
+	}
+}
+
+func TestService_RemoveCellV2(t *testing.T) {
+	type fields struct {
+		CellService chronograf.CellService
+	}
+	type args struct {
+		id string
+	}
+	type wants struct {
+		statusCode  int
+		contentType string
+		body        string
+	}
+
+	tests := []struct {
+		name   string
+		fields fields
+		args   args
+		wants  wants
+	}{
+		{
+			name: "remove a cell by id",
+			fields: fields{
+				&mocks.CellService{
+					DeleteCellF: func(ctx context.Context, id chronograf.ID) error {
+						if id == "2" {
+							return nil
+						}
+
+						return fmt.Errorf("wrong id")
+					},
+				},
+			},
+			args: args{
+				id: "2",
+			},
+			wants: wants{
+				statusCode: http.StatusNoContent,
+			},
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			s := &Service{
+				Store: &mocks.Store{
+					CellService: tt.fields.CellService,
+				},
+				Logger: log.New(log.DebugLevel),
+			}
+
+			r := httptest.NewRequest("GET", "http://any.url", nil)
+
+			r = r.WithContext(httprouter.WithParams(
+				context.Background(),
+				httprouter.Params{
+					{
+						Key:   "id",
+						Value: tt.args.id,
+					},
+				}))
+
+			w := httptest.NewRecorder()
+
+			s.RemoveCellV2(w, r)
+
+			res := w.Result()
+			content := res.Header.Get("Content-Type")
+			body, _ := ioutil.ReadAll(res.Body)
+
+			if res.StatusCode != tt.wants.statusCode {
+				t.Errorf("%q. RemoveCellV2() = %v, want %v", tt.name, res.StatusCode, tt.wants.statusCode)
+			}
+			if tt.wants.contentType != "" && content != tt.wants.contentType {
+				t.Errorf("%q. RemoveCellV2() = %v, want %v", tt.name, content, tt.wants.contentType)
+			}
+			if eq, _ := jsonEqual(string(body), tt.wants.body); tt.wants.body != "" && !eq {
+				t.Errorf("%q. RemoveCellV2() = \n***%v***\n,\nwant\n***%v***", tt.name, string(body), tt.wants.body)
+			}
+		})
+	}
+}
+
+func TestService_UpdateCellV2(t *testing.T) {
+	type fields struct {
+		CellService chronograf.CellService
+	}
+	type args struct {
+		id  string
+		upd chronograf.CellUpdate
+	}
+	type wants struct {
+		statusCode  int
+		contentType string
+		body        string
+	}
+
+	tests := []struct {
+		name   string
+		fields fields
+		args   args
+		wants  wants
+	}{
+		{
+			name: "update a cell",
+			fields: fields{
+				&mocks.CellService{
+					UpdateCellF: func(ctx context.Context, id chronograf.ID, upd chronograf.CellUpdate) (*chronograf.Cell, error) {
+						if id == "2" {
+							return &chronograf.Cell{
+								CellContents: chronograf.CellContents{
+									ID:   chronograf.ID("2"),
+									Name: "example",
+								},
+								Visualization: chronograf.V1Visualization{
+									Type: "line",
+								},
+							}, nil
+						}
+
+						return nil, fmt.Errorf("not found")
+					},
+				},
+			},
+			args: args{
+				id: "2",
```

Comment:
This test doesn't have an update body.  Perhaps add another test that does?

Also, does it make sense to be StatusOK on an update if there is body?